graph/schema.graphql

```diff
@@ -0,0 +1,52 @@
+type Document {
+  id: ID!
+  title: String!
+  key: String!
+  kind: String!
+  project: Project!
+}
+
+enum DocumentType {
+  VPAT
+  DesignSessionDeck
+  TRBLetter
+}
+
+input DocumentInput {
+  title: String!
+  key: String!
+  type: DocumentType!
+  project_id: ID!
+}
+
+type DocumentUpdateResponse {
+  success: Boolean!
+  message: String
+  document: Document!
+}
+
+type Project {
+  id: ID!
+  name: String!
+  documents: [Document!]!
+}
+
+input ProjectInput {
+  name: String!
+}
+
+type ProjectUpdateResponse {
+  success: Boolean!
+  message: String
+  project: Project!
+}
```

Comment:
When we chatted, we talked a bit about using the error pattern of making errors first class objects.  Again, I know these are early days, but you might consider exploring that for a generic error right now and adding more specific errors later.